<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Research Highlights</title>

<script src="site_libs/header-attrs-2.15/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/united.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>





<style type="text/css">
/* for pandoc --citeproc since 2.11 */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="styles.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-inverse  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Sotiris Tasoulis</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">
    <span class="fa fa-home"></span>
     
    Home
  </a>
</li>
<li>
  <a href="articles.html">
    <span class="fa fa-graduation-cap"></span>
     
    Publications
  </a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    <span class="fa fa-rocket"></span>
     
    Code
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="https://github.com/usersotiris/nonlinearclustering">Nonlinear Dimensionality Reduction for Clustering</a>
    </li>
    <li class="divider"></li>
    <li>
      <a href="https://github.com/panagiotisanagnostou/HiPart">HiPart: Hierarchical divisive clustering toolbox</a>
    </li>
  </ul>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Research Highlights</h1>

</div>


<div
id="developing-predictive-precision-medicine-models-by-exploiting-real-world-data-using-machine-learning-methods-theocharopoulos2024"
class="section level4">
<h4>Developing predictive precision medicine models by exploiting
real-world data using machine learning methods <span
class="citation">(Theocharopoulos et al., n.d.)</span></h4>
<p>Computational Medicine encompasses the application of Statistical
Machine Learning and Artificial Intelligence methods on several
traditional medical approaches, including biochemical testing which is
extremely valuable both for early disease prognosis and long-term
individual monitoring, as it can provide important information about a
person’s health status. However, using Statistical Machine Learning and
Artificial Intelligence algorithms to analyze biochemical test data from
Electronic Health Records requires several preparatory steps, such as
data manipulation and standardization. This study presents a novel
approach for utilizing Electronic Health Records from large, real-world
databases to develop predictive precision medicine models by exploiting
Artificial Intelligence. Furthermore, to demonstrate the effectiveness
of this approach, we compare the performance of various traditional
Statistical Machine Learning and Deep Learning algorithms in predicting
individuals’ future biochemical test outcomes. Specifically, using data
from a large real-world database, we exploit a longitudinal format of
the data in order to predict the future values of 15 biochemical tests
and identify individuals at high risk. The proposed approach and the
extensive model comparison contribute to the personalized approach that
modern medicine aims to achieve.</p>
</div>
<div
id="tarbase-v9.0-extends-experimentally-supported-mirna-gene-interactions-to-cell-types-and-virally-encoded-mirnas-10.1093nargkad1071"
class="section level4">
<h4>TarBase-v9.0 extends experimentally supported miRNA-gene
interactions to cell-types and virally encoded miRNAs <span
class="citation">(Skoufos et al. 2023)</span></h4>
<p>TarBase is a reference database dedicated to produce, curate and
deliver high quality experimentally-supported microRNA (miRNA) targets
on protein-coding transcripts. In its latest version (v9.0), it pushes
the envelope by introducing virally-encoded miRNAs, interactions leading
to target-directed miRNA degradation (TDMD) events and the largest
collection of miRNA-gene interactions to date in a plethora of
experimental settings, tissues and cell-types. It catalogues 6 million
entries, comprising 2 million unique miRNA-gene pairs, supported by 37
experimental (high- and low-yield) protocols in 172 tissues and
cell-types. Interactions are annotated with rich metadata including
information on genes/transcripts, miRNAs, samples, experimental contexts
and publications, while millions of miRNA-binding locations are also
provided at cell-type resolution. A completely re-designed interface
with state-of-the-art web technologies, incorporates more features, and
allows flexible and ingenioususe. The new interface provides the
capability to design sophisticated queries with numerous filtering
criteria including cell lines, experimental conditions, cell types,
experimental methods, species and/or tissues of interest. Additionally,
a plethora of fine-tuning capacities have been integrated to the
platform, offering the refinement of the returned interactions based on
miRNA confidence and expression levels, while boundless local retrieval
of the offered interactions and metadata is enabled.</p>
</div>
<div
id="neural-networks-voting-for-projection-based-ensemble-classifiers-10386944"
class="section level4">
<h4>Neural Networks Voting for Projection Based Ensemble Classifiers
<span class="citation">(P. Anagnostou et al. 2023)</span></h4>
<p>Ensemble learning has been proven effective in enhancing
classification accuracy by aggregating predictions from multiple base
classifiers. This paper introduces a novel approach to augmenting weak
projection-based classifiers using a Neural Network within a stacking
ensemble framework. The proposed method capitalizes on the diverse
strengths of both linear and complex models, harnessing the
interpretability of projection-based classifiers, while leveraging the
pattern recognition capabilities of Neural Networks. We present a
comprehensive algorithm involving dataset selection, preprocessing, base
model training, meta-feature generation, and Neural Network architecture
design and training. Extensive experiments demonstrate the efficiency of
our approach on a variety of high-dimensional biomedical datasets. Our
results showcase significant accuracy improvements over standalone
projection-based classifiers and conventional ensemble methods. We
analyze the interpretability of the hybrid ensemble, shedding light on
the insights drawn from its Neural Network component. This work not only
advances the field of ensemble learning, but also underscores the
potential of combining disparate classifier paradigms to achieve
superior predictive performance. The code for this study is
available1.1.https://github.com/panagiotisanagnostou/NNv-MRPV</p>
</div>
<div
id="two-phase-cooperative-learning-for-supervised-dimensionality-reduction-nellaspr"
class="section level4">
<h4>Two phase cooperative learning for supervised dimensionality
reduction <span class="citation">(Nellas et al. 2023)</span></h4>
<p>The simultaneous minimization of the reconstruction and
classification error is a hard non convex problem, especially when a
non-linear mapping is utilized. To overcome this obstacle, motivated by
the widespread success of Cooperative Neural Networks, an innovative
supervised dimensionality reduction framework is proposed, based on a
cooperative two phase optimization strategy. Specifically, the proposed
framework that requires minimal parameter adjustment consists of an
autoencoder for dimensionality reduction and a separator network for
separability assessment of the embedding. This scheme results in
meaningful and discriminable codes, which are optimized for the
classification task and are exploitable by any trainable classifier. The
experimental results showed that the proposed methodology achieved
competitive results against the state-of-the-art competing methods,
while being much more efficient in terms of parameter count. Finally, it
was empirically justified that the proposed methodology introduces
advanced behavioural explainability, while enabling applicability for
image generation tasks.</p>
</div>
<div
id="ensemble-clustering-for-boundary-detection-in-high-dimensional-data-10.1007978-3-031-53966-4_24"
class="section level4">
<h4>Ensemble Clustering for Boundary Detection in High-Dimensional Data
<span class="citation">(Panagiotis Anagnostou, Pavlidis, and Tasoulis
2024)</span></h4>
<p>The emergence of novel data collection methods has led to the
accumulation of vast amounts of unlabelled data. Discovering well
separated groups of data samples through clustering is a critical but
challenging task. In recent years various techniques to detect isolated
and boundary points have been developed. In this work, we propose a
clustering methodology that enables us to discover boundary data
effectively, discriminating them from outliers. The proposed methodology
utilizes a well established density based clustering method designed for
high dimensional data, to develop a new ensemble scheme. The
experimental results demonstrate very good performance, indicating that
the approach has the potential to be used in diverse domains.</p>
</div>
<div id="references" class="section level1">
<h1>References</h1>
<!-- <div id="refs"></div> -->
<!-- # Appendix -->
<div id="refs" class="references csl-bib-body hanging-indent">
<div id="ref-10.1007/978-3-031-53966-4_24" class="csl-entry">
Anagnostou, Panagiotis, Nicos G. Pavlidis, and Sotiris Tasoulis. 2024.
<span>“Ensemble Clustering for Boundary Detection in High-Dimensional
Data.”</span> In <em>Machine Learning, Optimization, and Data
Science</em>, edited by Giuseppe Nicosia, Varun Ojha, Emanuele La Malfa,
Gabriele La Malfa, Panos M. Pardalos, and Renato Umeton, 324–33. Cham:
Springer Nature Switzerland.
</div>
<div id="ref-10386944" class="csl-entry">
Anagnostou, P., P. Barmpas, S. K. Tasoulis, S. V. Georgakopoulos, and V.
P. Plagianakos. 2023. <span>“Neural Networks Voting for Projection Based
Ensemble Classifiers.”</span> In <em>2023 IEEE International Conference
on Big Data (BigData)</em>, 4567–74. Los Alamitos, CA, USA: IEEE
Computer Society. <a
href="https://doi.org/10.1109/BigData59044.2023.10386944">https://doi.org/10.1109/BigData59044.2023.10386944</a>.
</div>
<div id="ref-nellasPR" class="csl-entry">
Nellas, Ioannis A., Sotiris K. Tasoulis, Spiros V. Georgakopoulos, and
Vassilis P. Plagianakos. 2023. <span>“Two Phase Cooperative Learning for
Supervised Dimensionality Reduction.”</span> <em>Pattern
Recognition</em> 144: 109871. https://doi.org/<a
href="https://doi.org/10.1016/j.patcog.2023.109871">https://doi.org/10.1016/j.patcog.2023.109871</a>.
</div>
<div id="ref-10.1093/nar/gkad1071" class="csl-entry">
Skoufos, Giorgos, Panos Kakoulidis, Spyros Tastsoglou, Elissavet
Zacharopoulou, Vasiliki Kotsira, Marios Miliotis, Galatea Mavromati, et
al. 2023. <span>“<span class="nocase">TarBase-v9.0 extends
experimentally supported miRNA-gene interactions to cell-types and
virally encoded miRNAs</span>.”</span> <em>Nucleic Acids Research</em>
52 (D1): D304–10. <a
href="https://doi.org/10.1093/nar/gkad1071">https://doi.org/10.1093/nar/gkad1071</a>.
</div>
<div id="ref-Theocharopoulos2024" class="csl-entry">
Theocharopoulos, Panagiotis C., Sotiris Bersimis, Spiros V.
Georgakopoulos, Antonis Karaminas, Sotiris K. Tasoulis, and Vassilis P.
Plagianakos. n.d. <span>“Developing Predictive Precision Medicine Models
by Exploiting Real-World Data Using Machine Learning Methods.”</span>
<em>Journal of Applied Statistics</em>, 1–24. <a
href="https://doi.org/10.1080/02664763.2024.2315451">https://doi.org/10.1080/02664763.2024.2315451</a>.
</div>
</div>
</div>

<p> Copyright &copy; 2022, Inc. All rights reserved.</p>



</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
